mz_bokeh_package/environment.py

```diff
@@ -60,3 +60,19 @@ def get_error_page_url() -> str:
         mz_app_url = "error"
 
     return mz_app_url
+
+
+def get_webapp_host() -> str:
+    """get the web app host based on the environment.
+
+    Returns:
+        str: Web app host.
+    """
+    env = get_environment()
+
+    if env == "production":
```

Comment:
@roiweinreb to make it consistent with the other functions, please do first staging, then production, then dev (which returns the staging url) and then, as else, return error. If you think it's better to do it otherwise (e.g. throwing an exception in the "else" case, then please make the change consistently in all of them).